tests/manifests/__init__.py

```diff
@@ -163,30 +163,32 @@ def should_visit_with_values(
     ComponentDetails(
         name="postgres",
         has_ingress=False,
-        paths_consistency_noqa=("/docker-entrypoint-initdb.d/init-ess-dbs.sh"),
+        paths_consistency_noqa=("/docker-entrypoint-initdb.d/init-ess-dbs.sh", "/var/lib/postgres/data/pgdata"),
         is_shared_component=True,
     ),
     ComponentDetails(
         name="element-web",
         helm_key="elementWeb",
         has_service_monitor=False,
-        paths_consistency_noqa=("/etc/nginx/nginx.conf", "/etc/nginx/mime.types"),
+        paths_consistency_noqa=(
+            "/etc/nginx/nginx.conf",
+            "/etc/nginx/mime.types",
+            "/non-existant-so-that-this-works-with-read-only-root-filesystem",
+        ),
     ),
     ComponentDetails(
         name="matrix-authentication-service",
         helm_key="matrixAuthenticationService",
         has_db=True,
-        shared_component_names=(
-            "init-secrets",
-            "postgres",
-        ),
+        shared_component_names=("init-secrets", "postgres"),
     ),
     ComponentDetails(
         name="synapse",
         has_db=True,
         additional_values_files=[
             "synapse-worker-example-values.yaml",
         ],
+        paths_consistency_noqa=["/media/media_store"],
```

Comment:
Could we enhance the test_configs_and_mounts_setup to be aware of PVCs + emptyDirs?

Edit: oh there appears to be emptyDir support, so just PVCs are outstanding?